modules/cnv_install_1_4.adoc

```diff
@@ -44,7 +49,7 @@ the command.
 ====
 +
 ----
-$ ansible-playbook -i inventory -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml 
+$ ansible-playbook -i <inventory file> -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml 
```

Comment:
I'd prefer to have `inventory_file`. but most importantly, I miss the content of this file. I don't recall if @lukas-bednar provided you an example.

Reply:
we are not supposed to use all.yml, only cnv.yml

also, i believe that we should first cd into /usr/share/ansible/kubevirt-ansible
and execute the command from there in order to not need to be a privileged user

Reply:
I'm pretty sure we *do* need to specify all.yml. cnv.yml overrides only *some* of the vars there.

Reply:
@nellyc @lukas-bednar don't you use all.yml in QE?

Reply:
Update for the deploy command.
  - Remove all.yml extra_var
  - add apb_action=provision as an extra_var
```diff
-ansible-playbook -i <inventory file> -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml -e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml  /usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml
+ansible-playbook -i <inventory file> -e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml  /usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml -e apb_action=provision
```

Reply:
@ousleyp  please wait for QE approval before dropping all.yml.

Reply:
Dropping `all.yml` is wrong, regardless to that BZ. Why? upstream deployment of KubeVirt includes `all.yml`. `cnv.yml` has downstream-only modifications for it. I expect that most often than not, new variables can be set in all.yml without cnv.yml caring about them. If you drop `all.yml`, cnv.yml would have to become a not-so-precise copy of it. That's a very bad practice - it would quietly break when upstream is modified.

We *may* be able to include `all.yml` from within `cnv.yml` to help shorten the command line. I do not know ansible well enough to answer that.

Adding `apb_action=provision` is functionally harmless, but I don't see the benefit thereof.

Reply:
> the content of `inventory_file` is still not explained.

... or at least I cannot see it. You may say:

See "references" for an example `inventory_file` .

Reply:
I think the problem is that cnv.yml morphed into a patch of all.yml.  Things like `kubevirt_web_ui_branding` and `deploy_sriov_plugin` were duplicated in cnv.yml.  Unfortunately, downstream doesn't have a code review process, so now cnv.yml covers the functionality of all.yml.